src/components/Dashboard/Search.vue

```diff
@@ -383,8 +393,85 @@ export default class Search extends Vue {
 }
 </script>
 
-<style lang="scss" >
+<style lang="scss" scoped>
 @import '$assets/scss/theme.scss';
 @import '$assets/scss/search.scss';
-
+.button-search {
```

Comment:
we moved all css to search.scss, which can be used in both place (abc-auth and fas)